src/main/java/com/timgroup/statsd/StatsDProcessor.java

```diff
@@ -43,6 +43,8 @@
     protected volatile boolean shutdown;
     volatile boolean shutdownAgg;
 
+    protected String containerID;
```

Comment:
```suggestion
    private String containerID;
```

Reply:
Same here, it's `protected` so `TelemetryTest` can has access to it.